public/static/docs/command-reference/add.md

```diff
@@ -99,19 +99,19 @@ reproducible.
   analogous to using `git add` before `git commit`. Use `dvc commit` when ready
   to commit the results to cache.
 
+- `-f FILE`, `--file FILE` - specify name of the DVC-file it generates. This
```

Comment:
better to use use `path`

Reply:
Hmmm. But this comes from the core repo (command help output):

```console
$ dvc add -h
usage: dvc add [-h] [-q | -v] [-R] [--no-commit] [-f FILE]
               targets [targets ...]
```

Reply:
Discussing in https://github.com/iterative/dvc.org/pull/1029#issuecomment-594911780 and below.

Reply:
Resolving here.